week1_final/shopApp/cmd/server/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"yash/shopApp/config"
+	"yash/shopApp/internal/core/listeners/http"
+	"yash/shopApp/internal/customers"
+	cStore "yash/shopApp/internal/customers/store"
+	"yash/shopApp/internal/orders"
+	oStore "yash/shopApp/internal/orders/store"
+	"yash/shopApp/internal/products"
+	pStore "yash/shopApp/internal/products/store"
+	httpTransport "yash/shopApp/internal/transport/http"
+)
+
+func main() {
+	fmt.Println("Running...")
+	initDatabase()
+}
+
+func initDatabase() {
```

Comment:
rename the function name , this is doing more than initialising the database.

Reply:
renamed to `setupDatabase`